### PR TITLE
Connectagram: Update to Qt 6, switch to release version

### PR DESCRIPTION
**Pull request by win8linux**
This PR updates *Connectagram* to Qt 6, switches the recipe to a release version from a Git rev, and refreshes the recipe to current (as of time of writing) best practices.

Confirmed working on `x86_64`:
screenshot1
screenshot2
screenshot4
screenshot3

### CODE REVIEW

File: games-board/connectagram/connectagram-1.3.5.recipe [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Network$secondaryArchSuffix
+	lib:libQt6PrintSupport$secondaryArchSuffix
+	lib:libQt6Svg$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:libQt6Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
```
Comment (korli) [inline games-board/connectagram/connectagram-1.3.5.recipe:37]:
```suggestion

BUILD_REQUIRES="
```

File: games-board/connectagram/connectagram-1.3.5.recipe [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+features an online word lookup that fetches the definitions of each word \
+from Wiktionary. Your current progress is automatically saved."
+COPYRIGHT="2018-2024 Graeme Gott"
+HOMEPAGE="https://gottcode.org/connectagram/"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://gottcode.org/connectagram/connectagram-$portVersion.tar.bz2"
+CHECKSUM_SHA256="313598200733fd5c8080eb176d58cae407818825e7746030d4af00b3613ee454"
+SOURCE_FILENAME="connectagram-$portVersion.tar.gz"
+SOURCE_DIR="connectagram-$portVersion"
```
Comment (Begasus) [inline games-board/connectagram/connectagram-1.3.5.recipe:15]: SOURCE_DIR can be removed here also.

File: games-board/connectagram/connectagram-1.3.5.recipe [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+		-DCMAKE_INSTALL_MANDIR=$manDir
+
+	cmake --build build $jobArgs
+}
+
+INSTALL()
+{
+	cmake --install build
+	mv $appsDir/connectagram $appsDir/Connectagram
+	rm -rf $dataDir/Connectagram/{metainfo,icons,applications}
```
Comment (Begasus) [inline games-board/connectagram/connectagram-1.3.5.recipe:73]: Same here, sort alphabeticly :)

**Review by Begasus — APPROVED** [review]

**Comment by Begasus** [comment]
LGTM

**Pull request merged**